UnityProject/Assets/Scripts/Items/Weapons/ImplantExplosive.cs

```diff
@@ -27,11 +24,15 @@ protected override void Detonate()
 					//Macrobombs and microbombs will gib their victim
 					bodyPart.HealthMaster.OnGib();
 				}
-				else if (bodyPart.ContainedIn.BodyPartType == BodyPartType.Head)
+				else if (bodyPart.ContainedIn.BodyPartType != BodyPartType.Chest)
 				{
-					//Decapitates if explosive in head
+					//Removes limb it is implanted in
 					bodyPart.HealthMaster.DismemberBodyPart(bodyPart.ContainedIn);
 				}
+
+				//This prevents those with bomb proof armour from just tanking an explosion thats supposed to me inside them
+				bodyPart.HealthMaster.ApplyDamageToBodyPart(gameObject, 200, AttackType.Bomb, DamageType.Burn, bodyPart.ContainedIn.BodyPartType);
```

Comment:
you can use the internal attack type to bypass any armor. It also exists for everything that happens inside the body.